packages/dialog/src/base-types.ts

```
@@ -0,0 +1,37 @@
+import type {
+  DialogContentProps,
+  DialogOverlayProps,
+  DialogPortalProps,
+  DialogProps,
+} from '@radix-ui/react-dialog';
+import type { Prettify } from '@rn-primitives/types';
+
+type BaseDialogRootProps = Omit<Prettify<DialogProps>, 'children'>;
+
+type BaseDialogTriggerProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)